app/vmctl/vm_native.go

```diff
@@ -113,7 +229,15 @@ func (p *vmNativeProcessor) runWithFilter(ctx context.Context, f filter) error {
 		srcURL := fmt.Sprintf("%s/%s", p.src.addr, nativeExportAddr)
 		dstURL := fmt.Sprintf("%s/%s", p.dst.addr, nativeImportAddr)
 
-		return p.runSingle(ctx, f, srcURL, dstURL)
+		retryableFunc := func() error { return p.runSingle(ctx, f, srcURL, dstURL) }
+		attempts, err := p.backoff.Retry(ctx, retryableFunc)
+		p.s.Lock()
+		p.s.retries = attempts
```

Comment:
this should accumulate retries, not override them each time.

Reply:
fixed